src/scss/constants.scss

```diff
@@ -1,12 +1,18 @@
-$primaryColor: var(--primary-color);
-$backgroundColor: var(--background-color);
+$primary--over: var(--primary-color-deprecated);
+
+$accent--over: var(--accent--over);
+$accent--under: var(--accent--under);
+$content--over: var(--content--over);
+$content--under: var(--content--under);
+
+$primary--darker-20pp: var(--primary--darker-20pp);
+$primary--lighter-10pp: var(--primary--lighter-10pp);
```

Comment:
Not sure why these are added here while in 38 other places in 16 files it's just `var(--` without the need of `$constant-like-this`

Reply:
removed in 0cd2600f5d7e357895b206741557c96ce99cd816